.github/workflows/test-import.yaml

```diff
@@ -25,15 +25,15 @@ jobs:
 
       - name: Run test Go program that imports Zarf
         run: |
-          cd $(mktemp -d)
+          cd ..
+          mkdir test-import
+          cd test-import
           echo "$GO_MAIN" > main.go
           go mod init github.com/zarf-dev/test-import
-          go mod edit -replace github.com/zarf-dev/zarf=github.com/${{ github.repository }}@${COMMIT_SHA:0:12}
+          go mod edit -replace github.com/zarf-dev/zarf=../zarf
           go mod tidy
-          cat go.mod | grep -q ${COMMIT_SHA:0:12}
```

Comment:
We want to keep this test to make sure that the correct commit is being used in the go.mod file.

Reply:
Made sure 9f442cea1f989c62b0ec29ed6d8aded0f747f94e failed which is the commit before we fixed the issue last time.